### ISSUE TITLE
DIRECTORY_SIP_ACCOUNT - sip account not found 

### ISSUE THREAD

**Issue by jpawlowski** [amooma/GS5#6]
After a fresh installation and setup of a new phone and new SIP account the phone provisioning is successful. However the phone is not able to login with the provisioned SIP account as FreeSwitch does not know it:

```
2012-12-18 12:53:45.897492 [DEBUG] switch_cpp.cpp:1232 #C# [192.168.6.60] DIRECTORY_SIP_ACCOUNT - sip account not found - auth_name: a393928ca358e56657a2, domain: 192.168.6.60
```

Looking at the web interface it's exactly this auth_name that is existing. Something must have broken since the last commits...

**Comment by jpawlowski** [comment]
```
2012-12-18 16:32:57.297422 [NOTICE] switch_cpp.cpp:1232 #C# [192.168.6.60] SIP_ACCOUNT_QUERY - SELECT 
    `a`.`id`, 
    `a`.`uuid`, 
    `a`.`auth_name`, 
    `a`.`caller_name`, 
    `a`.`password`, 
    `a`.`voicemail_pin`, 
    `a`.`tenant_id`, 
    `a`.`sip_domain_id`, 
    `a`.`call_waiting`, 
    `a`.`clir`, 
    `a`.`clip`, 
    `a`.`clip_no_screening`, 
    `a`.`sip_accountable_type`,  
    `a`.`sip_accountable_id`, 
    `a`.`hotdeskable`, 
    `a`.`gs_node_id`, 
    `b`.`host` 
    FROM `sip_accounts` `a` JOIN `sip_domains` `b` ON `a`.`sip_domain_id` = `b`.`id`  
    WHERE `a`.`auth_name`= "a393928ca358e56657a2" AND `b`.`host` = "192.168.6.60" LIMIT 1
```

**Comment by jpawlowski** [comment]
Looks like config/database.yml has old content missing reference to MySQL...

**Comment by jpawlowski** [comment]
fixed by 4666adb72cd6a0316f0c1c338bc26449cebdd125
